### PR TITLE
fix(project-selector): Remove add project button for folks who cannot add a project

### PR DESCRIPTION
**Pull request by Chrissy**
disabled

### CODE REVIEW

**Review by billyvg — CHANGES_REQUESTED** [review]
Can we disable instead of hide?

**Comment by Chrissy** [comment]
This has been updated to be disabled instead of hidden.

File: src/sentry/static/sentry/app/components/projectSelector.jsx [hunk 1]
```diff
@@ -211,11 +210,14 @@ class ProjectSelector extends React.Component {
...
-          <Tooltip title="Add a project">
-            <AddButton to={`/organizations/${org.slug}/projects/new`} size="xsmall">
-              <StyledAddIcon src="icon-circle-add" /> project
-            </AddButton>
-          </Tooltip>
+          <AddButton
+            disabled={!hasProjectWrite}
+            to={`/organizations/${org.slug}/projects/new`}
+            size="xsmall"
+            title={hasProjectWrite ? null : 'You need project write access to use this'}
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/projectSelector.jsx]: `t()` - maybe "You don't have permission to add a project"?

File: src/sentry/static/sentry/app/components/projectSelector.jsx [hunk 2]
```diff
@@ -211,11 +210,14 @@ class ProjectSelector extends React.Component {
...
         emptyHidesInput
         inputActions={() => (
-          <Tooltip title="Add a project">
-            <AddButton to={`/organizations/${org.slug}/projects/new`} size="xsmall">
-              <StyledAddIcon src="icon-circle-add" /> project
-            </AddButton>
-          </Tooltip>
+          <AddButton
+            disabled={!hasProjectWrite}
+            to={`/organizations/${org.slug}/projects/new`}
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/projectSelector.jsx]: This should have a trailing slash I think?

**Review by billyvg — APPROVED** [review]

**Pull request merged**